packages/cashier/src/pages/withdrawal/crypto-withdraw-form/crypto-withdraw-form.tsx

```diff
@@ -153,7 +153,7 @@ const CryptoWithdrawForm = observer(() => {
                                 />
                             )}
                         </Field>
-                        <div>
+                        <div className='crypto-withdraw-form__percentage-container'>
```

Comment:
shouldnt it be like this `crypto-withdraw-form__percentage__container` ?

Reply:
from the existing code under this line it's `crypto-withdraw-form__percentage-selector` trying to make the class follow the existing lines below 🙏